docs/legal/dpa/subprocessors.rst

```diff
@@ -0,0 +1,62 @@
+Sub-Processor List
```

Comment:
We have a slightly out of date version of this list in our privacy policy: https://github.com/readthedocs/readthedocs.org/blob/master/docs/privacy-policy.rst#third-party-vendors

Should we attempt to unify these?

Reply:
Another good question. Does the format of the subprocessor list seem approachable enough that you'd want to link to it from the privacy policy? The sub-processor list does list commercial hosting providers -- ie Intercom, and no Cloudflare yet. Also, I'm not sure if we'd offer a DPA for community users or paid subscription users.

The sub-processor list will serve a very specific function. We're supposed to update it any time we add an additional external service, and we'll need to alert customers with signed DPA when adding a sub-processor.

It sounds like it should maybe be separate for now, but I don't have strong opinions here.

Reply:
I'll leave this as a future edit we can make, due to the fact that the subprocessor list is opinionated.